webpack.config.js

```diff
@@ -0,0 +1,55 @@
+var path = require('path');
+var NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
+
+module.exports = {
+    target: ['web', 'es5'],
+    entry: './lib/index.js',
+    output: {
+        path: path.resolve('./build'),
+        filename: 'plotly.js',
+        library: {
+            name: 'Plotly',
+            type: 'umd',
+            umdNamedDefine: true
+        }
+    },
+    module: {
+        rules: [{
+            test: /\.js$/,
+            exclude: /node_modules/,
+            use: {
+                loader: 'babel-loader'
+            }
+        }, {
+            test: /\.js$/,
+            include: /node_modules[\\\/](buffer|pngjs)[\\\/]/,
+            use: {
+                loader: 'babel-loader',
+                options: {
+                    babelrc: false,
+                    configFile: false,
+                    presets: [
+                        '@babel/preset-env'
+                    ]
+                }
+            },
+        }]
+    },
+    resolve: {
+        fallback: {
+            'stream': require.resolve('stream-browserify'),
+            'buffer': require.resolve('buffer/'),
+            'assert': require.resolve('assert/')
```

Comment:
These too - where are these needed? (It's OK if they are needed, I'm mostly just curious, also relevant for anyone who might include plotly.js from source, not dist bundles)

Reply:
Good call. `assert` and `buffer` are no longer needed. Addressed in https://github.com/plotly/plotly.js/pull/6355/commits/9fc59870eec4138d699709d839f5f0ee39d0cdeb.